lib/agent/backuprestore.js

```diff
@@ -112,9 +123,7 @@ function startRestore(params) {
     .assign(iaasConfiguration.max_retries ? {
       max_retries: iaasConfiguration.max_retries
     } : {})
-    .assign(_
-      .pick(iaasConfiguration, iaasSpecificParams[iaasConfiguration.name])
-    )
+    .assign(_.pick(iaasConfiguration, credHubParams))
```

Comment:
Please keep iaasSpecificParams here as well like [above](https://github.com/cloudfoundry-incubator/service-fabrik-blueprint-service/pull/32/files#diff-e93fa3dfc417c2664ad550ee7a369a8fR82) in case anybody requires.

Reply:
done